jenkins/ci.suse.de/cloud-mkcloud9-job-simpletest-s390x.yaml

```diff
@@ -0,0 +1,35 @@
+- job:
+    name: 'cloud-mkcloud9-job-simpletest-s390x'
```

Comment:
Do we need s390 testing? I didn't read anything about this in the PRD.

Reply:
thats a mistake in the PRD then I guess. I can remove it since the worker is down anyway.

Reply:
ok, SCPM tickets only talk about KVM. so removing..